src/components/datePicker/datePicker.tsx

```diff
@@ -0,0 +1,127 @@
+import ReactDatePicker, { registerLocale } from 'react-datepicker';
+import classNames from 'classnames/bind';
+import { FC, ReactNode, useMemo, useRef, forwardRef } from 'react';
+import { be, ru, enGB, es, uk, zhCN, Locale } from 'date-fns/locale';
+import { DatePickerHeader } from './header/datePickerHeader';
+import Input from './input/input';
+import styles from './datePicker.module.scss';
+
+const cx = classNames.bind(styles);
+
+const currentLanguageToLocale: { [key: string]: Locale } = {
+  en: enGB,
+  es,
+  be,
+  ru,
+  uk,
+  zh: zhCN,
+};
```

Comment:
All the translations managed in Service UI, so once the new language will be added, it will lead to changes in ui-kit.
On the other hand, some projects may require other locales to support, so it should be customizable.
So, let's store this object and use date-fns/locale within Service UI, and this component will just register locale come from props.